identity/app/views/noDiscussionsPage.scala.html

```diff
@@ -0,0 +1,19 @@
+@(  idRequest: services.IdentityRequest,
+    idUrlBuilder: services.IdentityUrlBuilder,
+    user: com.gu.identity.model.User,
+    activityType: String
+)(implicit request: RequestHeader, context: model.ApplicationContext)
+@import views.html.fragments.registrationFooter
+
+<div class="identity-wrapper monocolumn-wrapper identity-wrapper--stretched">
+    <div class="js-activity-stream activity-stream-content" data-user-id="@user.id" data-stream-type="@activityType"></div>
+    <h2>
+        No comments found for user
+    </h2>
+    <div>
+        Comments and replies will be shown on this page if a user has commented.
+        It appears this user does not exist or has not commented.
+        If this is your public profile, you can view your comments here once you have left your first comment.
```

Comment:
Do we need to get someone, such as a ux person/moderator/product manager to see if they're happy with this text/page?

Reply:
👍 I will confirm the copy with Jess

Reply:
`user` > `reader` if anything.